GUI/Forms/EntityInfoForm.cs

```diff
@@ -98,13 +98,31 @@ public void AddConnection(KVObject connectionData)
             var delay = connectionData.GetFloatProperty("m_flDelay");
             var timesToFire = connectionData.GetInt32Property("m_nTimesToFire");
 
+            string stimesToFire = "";
+
+            switch (timesToFire)
+            {
+                case 1:
+                    stimesToFire = "Only Once";
+                    break;
+                case 2:
```

Comment:
Why are we specializing 2 times into twice?

Reply:
Should I modify the following code again to make the two identical?
https://github.com/ValveResourceFormat/ValveResourceFormat/blob/25ac2246fed63e0321eaecef0e419e3fc0456aed/ValveResourceFormat/Resource/ResourceTypes/EntityLump.cs#L313-L314